### PR TITLE
(RE-4400) Add support for fedora 22

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by ScottGarman** [comment]
:+1:

File: manifests/mock/puppetlabs_mocks.pp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class rpmbuilder::mock::puppetlabs_mocks (
-  $fedora_releases  = ["14","15","16","17","18","19","20","21"],
+  $fedora_releases  = ["14","15","16","17","18","19","20","21","22"],
```
Comment (melissa) [inline manifests/mock/puppetlabs_mocks.pp]: Should I remove all but the currently supported mocks? Are we using fedora 14 for any of the network devices?

Comment (ScottGarman) [inline manifests/mock/puppetlabs_mocks.pp]: You know I think we do use F14 for one of the network platforms, though I'm drawing a blank at which one.

Comment (melissa) [inline manifests/mock/puppetlabs_mocks.pp]: I'll open a separate PR for removing the older  versions, and we can continue the conversation there

**Pull request merged**